<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffMainWindow</class>
 <widget class="QMainWindow" name="StaffMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plant Nursery - Staff Portal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>STAFF PORTAL</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="navigationLayout">
      <item>
       <widget class="QPushButton" name="plantManagementButton">
        <property name="text">
         <string>PLANT MANAGEMENT</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="staffManagementButton">
        <property name="text">
         <string>STAFF MANAGEMENT</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="navSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="text">
         <string>LOGOUT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="PlantManagementWidget" name="plantManagementWidget"/>
      <widget class="StaffManagementWidget" name="staffManagementWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlantManagementWidget</class>
   <extends>QWidget</extends>
   <header>PlantManagementWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StaffManagementWidget</class>
   <extends>QWidget</extends>
   <header>StaffManagementWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
